webpack.config.js

```diff
@@ -129,8 +129,8 @@ module.exports = {
 			name: "Kapta",
 			short_name: "Kapta",
 			lang: "en-GB",
-			theme_color: "#9f1203",
-			background_color: "#9f1203",
+			theme_color: " #1ca44c",
```

Comment:
to match the colour in the app, it's 1ca64f
```suggestion
			theme_color: " #1ca64f",
```